kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/RawCustomResourceOperationsImpl.java

```diff
@@ -346,9 +360,6 @@ public RawCustomResourceOperationsImpl dryRun(boolean isDryRun) {
    * @throws IOException in case of network/serialization failures or failures from Kubernetes API
    */
   public Map<String, Object> edit(String namespace, String name, String objectAsString) throws IOException {
-    // Append resourceVersion in object metadata in order to
-    // avoid : https://github.com/fabric8io/kubernetes-client/issues/1724
```

Comment:
This is already implemented here:
https://github.com/fabric8io/kubernetes-client/blob/6e439a876f3b7973c744aa35bece70aca4700f39/kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/RawCustomResourceOperationsImpl.java#L882